File: src/Tests.cs/YogurtTests.cs
using FluentAssertions;
using NUnit.Framework;
using TLK.LoveDeathFSharp.Kernel;

namespace TLK.LoveDeathFSharp.Tests
{
    internal class YogurtTests
    {
        [Test]
        public void YogurtRequestIsWeWantOhio()
        {
            // arrange
            var yogurt = new Yogurt();

            // assert
            Assert.That(yogurt.Request, Is.EqualTo("We. Want. Ohio."));
        }

        [Test]
        public void YogurtRequestShouldBeWeWantOhio()
        {
            // arrange
            var yogurt = new Yogurt();

            // assert
            yogurt.Request.Should().Be("We. Want. Ohio.");
        }
    }
}

File: src/Tests.fs/YogurtTests.fs
module TLK.LoveDeathFSharp.Tests.Yogurt

open NUnit.Framework
open FsUnit.TopLevelOperators
open TLK.LoveDeathFSharp.Kernel

let request (yogurt : Yogurt) = yogurt.Request

[<Test>]
let ``Yogurt's request should be "We. Want. Ohio."`` () =
    () |> Yogurt |> request |> should equal "We. Want. Ohio."
